app/routes/next/organization/$slug/settings.tsx

```diff
@@ -0,0 +1,186 @@
+    where: { slug: params.slug },
+    select: {
+      name: true,
+      slug: true,
+    },
+  });
+
+  invariantResponse(organization !== null, "Organization not found", {
+    status: 404,
+  });
+
+  return json({ organization });
+}
+
+const i18nNS = ["routes/next/organization/settings"];
+export const handle = {
+  i18n: i18nNS,
+};
+
+function createNavLinks(t: TFunction) {
+  return [
+    { to: "./general", label: t("links.general") },
+    { to: "./organize", label: t("links.organize") },
+    { to: "./web-social", label: t("links.webSocial") },
+    { to: "./admins", label: t("links.admins") },
+    { to: "./team", label: t("links.team") },
+    { to: "./danger-zone", label: t("links.dangerZone"), variant: "negative" },
+  ];
+}
+
+function Settings() {
+  const loaderData = useLoaderData<typeof loader>();
+  const location = useLocation();
+  const [searchParams] = useSearchParams();
+  const { t } = useTranslation(i18nNS);
+
+  const navLinks = createNavLinks(t);
+
+  const deep = searchParams.get("deep");
+
+  const menuClasses = classNames(
+    "mv-w-full @md:mv-w-1/3 @2xl:mv-w-1/4 mv-max-h-screen @md:mv-max-h-fit mv-flex mv-flex-col mv-absolute @md:mv-relative mv-top-0 mv-bg-white @md:mv-border-l @md:mv-border-b @md:mv-rounded-bl-xl @md:mv-self-start",
+    deep !== null && deep !== "false" && "mv-hidden @md:mv-block"
+  );
+
+  const outletClasses = classNames(
+    "mv-overflow-hidden @md:mv-w-2/3 @2xl:mv-w-3/4 @md:mv-border-x @md:mv-border-b @md:mv-rounded-b-xl @md:mv-mb-4 @lg:mv-mb-24 mv-bg-white",
+    (deep === null || deep === "false") && "mv-hidden @md:mv-block"
+  );
+
+  return (
+    <div className="mv-w-full mv-max-w-none mv-px-0 mv-mx-auto @md:mv-px-4 @md:mv-max-w-screen-container-md @lg:mv-max-w-screen-container-lg @xl:mv-max-w-screen-container-xl @xl:mv-px-6 @2xl:mv-max-w-screen-container-2xl @md:mv-mt-2">
+      <div className="mv-hidden @md:mv-block mv-mb-8">
+        <div className="mv-flex mv-flex-col mv-gap-8 @lg:mv-gap-14">
+          <TextButton weight="thin" variant="neutral" arrowLeft>
+            <Link
+              to={`/organization/${loaderData.organization.slug}`}
+              prefetch="intent"
+            >
+              {t("content.back")}
+            </Link>
+          </TextButton>
+          <h3 className="mv-mb-0 mv-font-bold">{t("content.edit")}</h3>
+        </div>
+      </div>
+      <div className="mv-hidden @md:mv-block">
+        <Section variant="primary" withBorder>
+          <Section.Header>{loaderData.organization.name}</Section.Header>
+        </Section>
+      </div>
+      <div className="mv-w-full @md:mv-flex @md:mv-mb-20 @lg:mv-mb-0">
+        <div className={menuClasses}>
+          <div className="mv-flex mv-gap-2 mv-items-center mv-justify-between @md:mv-hidden">
+            <span className="mv-p-6">
+              <h1 className="mv-text-2xl mv-m-0">{t("content.settings")}</h1>
+            </span>
```

Comment:
The length of the heading in combination with the padding on it and the padding on the "X" Button leads to container overflow on mobile (width: 375, IPhone SE 2nd gen.).

Proposal:
The "X" Button already has px-4. Maybe we change the padding of the headline from "p-6" to "py-6 pl-6 pr-0"?